tools/mkmodel/mkmodel.c

```diff
@@ -1100,9 +1120,93 @@ int read_anim_channel(cgltf_data *data, cgltf_animation_channel *in_channel, glt
         fprintf(stderr, "Error: failed reading channel output accessor.\n");
         return 1;
     }
+    out_channel->output_min[0] = FLT_MAX;
+    out_channel->output_min[1] = FLT_MAX;
+    out_channel->output_min[2] = FLT_MAX;
+    out_channel->output_min[3] = FLT_MAX;
+    out_channel->output_max[0] = -FLT_MAX;
+    out_channel->output_max[1] = -FLT_MAX;
+    out_channel->output_max[2] = -FLT_MAX;
+    out_channel->output_max[3] = -FLT_MAX;
```

Comment:
I think these should be `FLT_MIN`

Reply:
It should not be FLT_MIN since output_max could be less than 0 and FLT_MIN is approximately 0 rather than -2^(127).